esmvaltool/diag_scripts/quantilebias/quantilebias.R

```diff
@@ -0,0 +1,154 @@
+# #############################################################################
+# quantilebias.r
+# Authors:       E. Arnone (ISAC-CNR, Italy)
+# 	         S. Terzago (ISAC-CNR, Italy)
+# 	         J. von Hardenberg (ISAC-CNR, Italy)
+# #############################################################################
+# Description
+# ESMValTool diagnostic for calculation of the precipitation quantile bias
+# following Mehran et al. (2014)
+#
+# Required
+# - #CDO
+# - observational monthly mean global precipitation climatology
+# - (e.g. from the GPCP project: http://gpcp.umd.edu)
+#
+# Optional
+#
+# Caveats
+#
+# Modification history
+#    20180926-A_arno_en: Refined for usage as recipe
+#    20180518-A_arno_en: Written for v2.0
+#
+# #############################################################################
+
+library(tools)
+library(yaml)
+
+cdo <- function(command, args="", input="", options="", output="") {
+  if (args != "") args <- paste0(",", args)
+  argstr <- paste0(options, " ", command, args, " ", input, " ", output)
```

Comment:
The input and output files need to be quoted, in case there are e.g. spaces or other special characters somewhere in the path.
```suggestion
  argstr <- paste0(options, " ", command, args, " '", input, "' '", output, "'")
```